sql/views.sql

```diff
@@ -113,9 +113,11 @@ SELECT cagg.user_view_schema AS view_schema,
   cagg.materialized_only,
   ht.schema_name AS materialization_hypertable_schema,
   ht.table_name AS materialization_hypertable_name,
+  format('%I.%I', raw_ht.schema_name, raw_ht.table_name)::regclass AS raw_hypertable,
```

Comment:
We decided previously to not use regclass in views, as it had some undesirable properties when joining with other views and catalog info. This is also not consistent with other views that use `hypertable_schema` and `hypertable_name`.

IMO, we should have two field at the start of the view which is `hypertable_schema` and `hypertable_name` similar to what we have in, e.g., the `chunks`,`dimensions`, and `compression_settings` views.

Reply:
Thank you for reminding about this decision. I can clearly see that it is nightmare for usability to use the pair of strings.

Reply:
'<table_name>'::regclass returns the OID , if you need to pass the oid as arg to a function.

Reply:
@gayyappan It is necessary to concatinate the schema name to it to avoid unexpected ambiguity, right?